server/app/views/admin/questions/QuestionEditView.java

```diff
@@ -509,4 +496,22 @@ private DivTag repeatedQuestionInformation() {
             Styles.BORDER_4,
             Styles.BORDER_BLUE_400);
   }
+
+  private DivTag administrativeNameField(String adminName, boolean editExistingQuestion) {
+    if (editExistingQuestion) {
+      return div()
+          .withClass(Styles.MB_2)
+          .with(
+              p("Administrative identifier. This value can't be changed")
+                  .withClasses(BaseStyles.INPUT_LABEL),
+              p(adminName).withId("question-name-input").withClasses(BaseStyles.FORM_FIELD),
+              input().isHidden().withName(QUESTION_NAME_FIELD).withValue(adminName));
+    }
+    return FieldWithLabel.input()
+        .setId("question-name-input")
+        .setFieldName(QUESTION_NAME_FIELD)
+        .setLabelText("Administrative identifier. This value can't be changed later*")
```

Comment:
Does * signify that it's mandatory?

Reply:
Correct. There was a push to add a "*" to all required fields.